build/optaplanner-bom/pom.xml

```diff
@@ -479,19 +479,6 @@
         <type>test-jar</type>
         <version>${version.org.optaplanner}</version>
       </dependency>
-      <!-- distribution -->
-      <dependency>
-        <groupId>org.optaplanner</groupId>
-        <artifactId>optaplanner-docs</artifactId>
-        <type>zip</type>
-        <version>${version.org.optaplanner}</version>
-      </dependency>
-      <dependency>
-        <groupId>org.optaplanner</groupId>
-        <artifactId>optaplanner-distribution-internal</artifactId>
```

Comment:
I think the distribution should move to the `dependencyManagement` of the build parent as the Quickstarts depend on it:
https://github.com/kiegroup/optaplanner-quickstarts/blob/stable/build/optaplanner-distribution/pom.xml#L43

Reply:
Indeed, that's what I'm doing in the downstream PR.

Reply:
That will work. On the other hand, it could go into `optaplanner-build-parent` here, as the quickstarts distribution module, opposed to all individual quickstarts, depends on it.

Reply:
Could be, but do I need it? The only place that does depend on it is the distribution, and I think it should stay like that.